Roys/app/src/main/java/com/dwg_karrier/roys/DataBaseOpenHelper.java

```diff
@@ -76,8 +76,12 @@ public void setWordCount(String url, int wordCount) {
 
   public void insertScriptedData(String url) {
     SQLiteDatabase dataBase = getWritableDatabase();
-    dataBase.execSQL("INSERT INTO PAGE (URL) VALUES ('" + url + "');");
-    dataBase.close();
+    try {
+      dataBase.execSQL("INSERT INTO PAGE (URL) VALUES ('" + url + "');");
+      dataBase.close();
+    } catch (Exception exception) {
+      dataBase.close();
+    }
```

Comment:
in google code style, don't make empty block 
https://google.github.io/styleguide/javaguide.html#s4.1.2-braces-empty-blocks

I want to checking url duplication by try-catch.. 
how to change this line looks good? Add failed log? or I think get all url list and check duplication.